src/elfpy/markets.py

```diff
@@ -233,13 +233,13 @@ def open_short(
         )
         # amount to cover the worst case scenario where p=1. this amount is 1-p. see logic above.
         max_loss = trade_amount - trade_result.user_result.d_base
-        wallet_deltas = Wallet(
+        agent_deltas = Wallet(
```

Comment:
If you want to make this change (fine with me) then you should do it everywhere. Here is what I found with grep:
`./src/elfpy/markets.py:        wallet_deltas = Wallet(
./src/elfpy/markets.py:        return market_deltas, wallet_deltas
./src/elfpy/simulators.py:                wallet_deltas = self.market.trade_and_update(trade)
./src/elfpy/simulators.py:                agent.update_wallet(wallet_deltas, self.market)
./src/elfpy/simulators.py:                    wallet_deltas.__dict__,
./src/elfpy/agent.py:    update_wallet(self, wallet_deltas: Wallet, market: Market) -> None
./src/elfpy/agent.py:    def update_wallet(self, wallet_deltas: Wallet, market: Market) -> None:
./src/elfpy/agent.py:        wallet_deltas : Wallet
./src/elfpy/agent.py:        for key, value_or_dict in wallet_deltas.__dict__.items():`

Reply:
regular expressions to the rescue! changed the first 5. I'm gonna keep the last 4, cus those are inside the agent, and the agent is managing deltas on its wallet, not on itself. in the other classes, there are `market_deltas` and `agent_deltas` which both refer to wallets, so we need that extra qualifier.

Reply:
following up after reviewing the rest of the PR -- looks like it's good! Resolve if you agree.